x-pack/plugins/observability_solution/infra/public/pages/logs/settings/indices_configuration_panel.tsx

```diff
@@ -54,6 +75,32 @@ export const IndicesConfigurationPanel = React.memo<{
     });
   }, [indicesFormElement, trackChangeIndexSourceType]);
 
+  useEffect(() => {
+    const getNumberOfInfraRules = async () => {
+      if (http) {
+        const { ruleExecutionStatus } = await loadRuleAggregations({
+          http,
+          typesFilter: [LOG_THRESHOLD_ALERT_TYPE_ID],
+        });
+        const numberOfRules = Object.values(ruleExecutionStatus).reduce(
+          (acc, value) => acc + value,
+          0
+        );
+        setNumberOfLogsRules(numberOfRules);
+      }
+    };
+    getNumberOfInfraRules();
+  }, [http]);
+
+  useEffect(() => {
+    const getLink = async () => {
+      const resLink = await rulesLocator?.getUrl({ type: [LOG_THRESHOLD_ALERT_TYPE_ID] });
+      setViewAffectedRulesLink(resLink);
+    };
+
+    getLink();
+  }, [rulesLocator]);
```

Comment:
**suggestion:** You could simplify this part with the `useUrl` hook from the locator service, which executes the same logic implemented now.
```tsx
const viewAffectedRulesLink = rulesLocator.useUrl({ type: [LOG_THRESHOLD_ALERT_TYPE_ID] })
```

The same approach applies for the link on the metrics settings.

Reply:
@tonyghiani  Thanks for the suggestion. Updated in 276668a2db1154cfcc1bae9795d3d6b63272eca9